src/rebar3_hex.erl

```diff
@@ -11,6 +11,8 @@
         , help_opt/0
         ]).
 
+-export([to_atom/1, to_list/1, to_binary/1]).
```

Comment:
Rationale : the less we rely on rebar3 internal util modules, the less burden it is for rebar3 to make changes without worrying about breaking the rest of the world.